Code/Framework/AzCore/Platform/Android/AzCore/Android/AndroidEnv.h

```diff
@@ -85,9 +85,6 @@ namespace AZ
             //! \return A pointer to the JNIEnv on the current thread.
             JNIEnv* GetJniEnv() const;
 
-            //! Request the global reference to the activity java virtual machine
```

Comment:
Is this removal related to the new PostFrame function?

Reply:
It's related to how OpenXR gets the JavaVM. I added that getter for OpenXR to use it, but that addition was wrong since I learnt that the right way to obtain the JavaVM is to do `GetJniEnv()->GetJavaVM()`. More details in the description.